provider/resources.go

```diff
@@ -32,7 +32,7 @@ var metadata []byte
 // Provider returns additional overlaid schema and metadata associated with the provider..
 func Provider() tfbridge.ProviderInfo {
 	info := tfbridge.ProviderInfo{
-		P:                 pf.ShimProvider(shim.NewProvider()),
+		P:                 pf.ShimProvider(talos.New()),
```

Comment:
❓ For my own understanding, is `pf.ShimProvider` still needed even thought upstream is no longer shimmed?

Reply:
@UnstoppableMango Yes, the type for `P` is a Pulumi type, not a TF provider type. So we must wrap the TF provider still in a Pulumi type.